whois-commons/src/main/java/net/ripe/db/whois/common/dao/OutgoingMessageDao.java

```diff
@@ -30,11 +30,4 @@ public List<String> getEmails(final String messageId) {
                 "SELECT email from outgoing_message where message_id = ?",
                 String.class, messageId);
     }
-
-    public boolean isEmailExists(final String messageId, final String email){
-        return Boolean.TRUE.equals(jdbcTemplate.query("select email from outgoing_message where message_id = ? and email = ?",
```

Comment:
Isn't this WHERE clause case-insensitive already?

Reply:
yes, it is. I will change the name of the PR...In the end is just a little refactor to remove the logic from Dao

Reply:
Is this what we want? We could leave the SELECT to match the message-id and email address